src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.AddressBook;
+
+import static java.util.Objects.requireNonNull;
+
+/*
+* sorts the addressbook
```

Comment:
Use a Javadoc header comment. right now your header comment will not get picked up as javadoc header comment. add a more detailed comment. eg. sorts by 'X' and return a lit of Y